src/bundle/Resources/views/content/locationview.html.twig

```diff
@@ -132,6 +135,7 @@
         'bundles/ezplatformadminui/js/scripts/admin.location.add.custom_url.js'
         '@EzPlatformAdminUiBundle/Resources/public/js/scripts/button.state.toggle.js'
         '@EzPlatformAdminUiBundle/Resources/public/js/scripts/admin.version.edit.conflict.js'
+        'bundles/ezplatformadminui/js/scripts/admin.location.bookmark.js'
```

Comment:
Which form should we use? 'bundles/ezplatformadminui/js/scripts/admin.location.bookmark.js' or '@EzPlatformAdminUiBundle/Resources/public/js/scripts/admin.location.bookmark.js'?

Reply:
@mikadamczyk @ViniTou I'd rather use `@EzPlatformAdminUiBundle` as we use resource names for twig templates.